app/components/shared/ImageUploadField.js

```diff
@@ -3,6 +3,7 @@
 import React from 'react';
 import classNames from 'classnames';
 import styled from 'styled-components';
+import loadImage from 'blueimp-load-image';
```

Comment:
Before we go live into prod maybe we move this into a dynamic require / bundle. We could even load it only when the first drag+drop starts, and don't finish the existing "uploading" animation until it's loaded, so it's mostly invisible?

Reply:
Done! We’ll now load it when you first drag over the document or drag zone (as sometimes you can skip the document! Oops!) and we set the “uploading” animation early.

We also start lazy-loading when you click the “upload” buttons, which means it loads while you’re browsing! Super schmick! 👌🏼

Reply:
That's really awesome! Nice work!!